perl/lib/BarnOwl/Message/Zephyr.pm

```diff
@@ -133,6 +144,14 @@ sub zephyr_cc {
     return undef;
 }
 
+sub zephyr_cc_without_recipient {
+    my $self = shift;
+    my $recipient = lc(strip_realm($self->recipient));
+    my $cc = $self->zephyr_cc;
+    return grep { lc(strip_realm($_)) ne $recipient } split(/\s+/, $cc) if defined $cc;
```

Comment:
I added
```perl
# Note: This is the cc-line without the recipient; it does not include
# the sender.
```
above the definition of the function; is that sufficient, or did you want something more specific?